interpreter/js/src/main/scala/sigmastate/js/Interpreter.scala

```diff
@@ -0,0 +1,7 @@
+package sigmastate.js
+
+import scala.scalajs.js.annotation.JSExportTopLevel
+
+@JSExportTopLevel("Interpreter")
+object Interpreter {
```

Comment:
ScalaDoc missed

Reply:
removed in this PR, will be added later.